index.html

```diff
@@ -209,14 +211,29 @@ <h3>Skills</h3>
         
                     </div>
                     
-                    <div class="form-contact">
+                    <div class="form-contact-mobile">
                         <form class="contact-form" action="https://formspree.io/f/xayknejl" method="post">
                             <input type="text" name="name" maxlength="30" placeholder="Full Name" required>
                             <input type="email" name="email" placeholder="Email address" required>
                             <textarea placeholder="Enter text here" rows="7" maxlength="500" required></textarea>
                             <button type="submit" class="buttom-contact-form">Get in touch</button>
                         </form>
                     </div>
+                    <div class="form-contact-desktop">
+                        <form class="contact-form2" action="https://formspree.io/f/xayknejl" method="post">
+                            <div class="names">
+                                <div class="name-text"><input type="text" name="name" maxlength="30" placeholder="First Name" required></div>
+                                <div class="last-text"><input type="text" name="lastname" maxlength="30" placeholder="Last Name" required></div>
+                                
+                            </div>
+                            <div class="emails">
+                                <input type="email" name="email" placeholder="Email address" required>
+                            </div>
+                            
+                            <textarea placeholder="Type something" rows="7" maxlength="500" required></textarea>
+                            <button type="submit" class="buttom-contact-form2">Get in touch</button>
```

Comment:
Great design @taldr27 🤝

- [ ] [Optional] Kindly implement all button interactions of `hover` and `enable` across all the buttons in your design.